reactor-core/src/jcstress/java/reactor/core/publisher/SinkManyBestEffortStressTest.java

```diff
@@ -0,0 +1,217 @@
+/*
+ * Copyright (c) 2020-Present Pivotal Software Inc, All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package reactor.core.publisher;
+
+import org.openjdk.jcstress.annotations.Actor;
+import org.openjdk.jcstress.annotations.Arbiter;
+import org.openjdk.jcstress.annotations.JCStressTest;
+import org.openjdk.jcstress.annotations.Outcome;
+import org.openjdk.jcstress.annotations.State;
+import org.openjdk.jcstress.infra.results.I_Result;
+import org.openjdk.jcstress.infra.results.LI_Result;
+import org.reactivestreams.Subscription;
+
+import static org.openjdk.jcstress.annotations.Expect.*;
+
+public abstract class SinkManyBestEffortStressTest {
```

Comment:
if there is no shared code than the abstract class can be avoided (it is not required by JCstress or anything like that)

Reply:
the `abstract` keyword you mean? I still need inner classes for individual tests I guess ?

Reply:
you can as well have N classes/files, unless you want to group them, of course

Reply:
yeah, I want to group the stress tests here